ff-parser/src/main/java/org/uniprot/core/flatfile/parser/impl/ft/FtLineObject.java

```diff
@@ -24,15 +24,63 @@ public EvidenceInfo getEvidenceInfo() {
     }
 
     public static class FT {
-        public FTType type;
+        private FTType type;
 
         // can have fuzzy type thus cannot use int.
-        public String location_start;
-        public String location_end;
+        private String location_start;
```

Comment:
not part of the task, so, very very very optional, is it possible to change it to camelCase?

Reply:
https://www.ebi.ac.uk/panda/jira/browse/TRM-23567